app/views/catalog/_export_button.html.erb

```diff
@@ -20,5 +20,9 @@
         <%= link_to content_tag('i', '', :class => 'icon-file-text-o') + t('ddf.tools.ris'), catalog_path(document, :format => 'ris') %>
       </li>
     <% end %>
+
+    <li>
+      <%= link_to t('ddf.tools.email_result'), citations_new_path(id: document.id), data: { ajax_modal: 'trigger'} %>
```

Comment:
probably would be better alright, 'citations_new` was the helper generated by Rails from my route, but I can manually configure something more appropriate.